app/core/chat/dst/intent_detection.py

```diff
@@ -17,26 +17,26 @@ def recognize_intent(self, *args, **kwargs) -> IntentEnum:
 class ChatGPTBasedIntentDetectionModule(AbstractIntentDetectionModule):
     id_prompt: str = "Recognize the intent from list [{0}]" ' for a given utterance: "{1}".'
 
-    def __init__(self, chatgpt_handler: ChatGPTHandler):
+    def __init__(self, chatgpt_handler: ChatGPTBridge):
         self.chatgpt_handler = chatgpt_handler
 
-    def fuzzy_intent_matching(self, response: str, intents: List[str]) -> IntentEnum:
-        current_intent: str = intents[0]
-        max_score: int = 0
-        for intent in intents:
-            score: int = fuzz.ratio(response, intent)
-            if score > max_score:
-                current_intent = intent
-                max_score = score
-        return IntentEnum(current_intent)
-
     def recognize_intent(self, message: str) -> IntentEnum:
-        intents: List[str] = [intent.value for intent in IntentEnum]
-        prompt: str = self.id_prompt.format(", ".join(intents), message)
+        intents: str = ", ".join(f"{intent.name} ({intent.value})" for intent in IntentEnum)
```

Comment:
Sure, as long as you tested it in multiple scenarios and it works fine. IMO it is even better, but I tested the previous version and it worked for me.